### ISSUE TITLE
Use `Int64 / C` as default

### ISSUE THREAD

**Issue by MilesCranmer** [SymbolicML/DynamicQuantities.jl#20]
cc @oscardssmith 

I think it would probably be much faster to use `Int64 / C`, for some choice of constant `C`, than to use `Rational{Int16}` as is currently done – as it would avoid repeated `gcd` calls.

We would want to pick some constant `C` such that most rationals expressed by `Rational{Int16}` that people would typically want to use could also be expressed this way.

**Comment by MilesCranmer** [comment]
What about, for example,
```
C = 2^8 * 3^6 * 5^4 * 7^2 * 11^2 * 13 * 15  # = 2385171360000
```
meaning you could represent any fraction which can be built by a combination of those powers. But you could not represent, say, 1/(15^2). But you could get awfully close.

**Comment by oscardssmith** [comment]
I really think the default should probably be `Int8/6`. There are some cases where you need finer granularity than that, but they're pretty rare and I don't think they should be the default.

**Comment by MilesCranmer** [comment]
(Fixed by #21 but forgot to close this)
